src/ECER.Clients.RegistryPortal/ECER.Clients.RegistryPortal.Server/Applications/ApplicationsEndpoints.cs

```diff
@@ -119,11 +120,13 @@ public enum CertificationType
 
 public enum PortalStage
 {
+  CertificationType,
+  Declaration,
   ContactInformation,
   Education,
-  References,
+  CharacterReferences,
```

Comment:
super nitpik. I think we only have 1 character reference so this can be singular?

Reply:
Agreed that would be a better field name but going to leave this for now to match the dynamics field name `ecer_characterreferences`